src/components/Reactions/ReactionTooltipContent.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import {withPersonalDetails} from '../OnyxProvider';
+import getPersonalDetailsByIDs from '../../libs/getPersonalDetailsByIDs';
+import Text from '../Text';
+
+const propTypes = {
+    /**
+     * A list of emoji codes to display in the tooltip.
+     */
+    emojiCodes: PropTypes.arrayOf(PropTypes.string).isRequired,
+
+    /**
+     * The name of the emoji to display in the tooltip.
+     */
+    emojiName: PropTypes.string.isRequired,
+
+    /**
+     * A list of account IDs to display in the tooltip.
+     */
+    accountIDs: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+const ReactionTooltipContent = (props) => {
+    const users = getPersonalDetailsByIDs(props.accountIDs);
```

Comment:
Am I correct that this will no longer keep the order in which reactions were added, it will actually be alphabetical based on email? 

If so, we need to sort this list based on the accountIDs prop

Reply:
Actually thinking about this more, I think we should update the `getPersonalDetailsByIDs` method to not use `_.filter` and write our own method instead that directly puts the details into the return array where they need to go. Something like this, which should have the same runtime as the current method, but preserve the order the accountIDs were sent in:

```
const result = [];
_.each(personalDetails, detail => {
    for (i=0; i < accountIDs.length; i++) {
        if (detail.accountID === accountIDs[i]) {
            result[i] = detail;
            break;
        }
    }
});
```

Reply:
oh yeah, that's better, you are right